test/e2e/tools/event-writer/install-ebpf-xdp.ps1

```diff
@@ -0,0 +1,739 @@
+      Expand-Archive -Path "$LocalPath\Build-x64-native-only.NativeOnlyRelease.zip" -DestinationPath "$LocalPath\Build-x64-native-only.NativeOnlyRelease\msi" -Force
+      Copy-Item "$LocalPath\Build-x64-native-only.NativeOnlyRelease\msi\Build-x64-native-only NativeOnlyRelease\*.msi" -Destination $LocalPath
+
+      Start-Process -FilePath "$($env:WinDir)\System32\MSIExec.exe" -ArgumentList @("/i", "$LocalPath\ebpf-for-windows.msi", "/qn", "INSTALLFOLDER=`"$($env:ProgramFiles)\ebpf-for-windows`"", "ADDLOCAL=eBPF_Runtime_Components") -PassThru | Wait-Process
+      If(-Not (Assert-SoftwareInstalled -ServiceName:'eBPFCore' -Silent) -Or
+         -Not (Assert-SoftwareInstalled -ServiceName:'NetEbpfExt' -Silent))
+      {
+         Write-Error -Message:"`teBPF service failed to install"
+         Throw
+      }
+
+      $isSuccess = Assert-SoftwareInstalled -ServiceName:"eBPFCore"
+   }
+   Catch
+   {
+      $isSuccess = $false
+      Write-Host "EBPF install failed : $_"
+      Uninstall-eBPF
+   }
+
+   Return $isSuccess
+}
+
+<#
+ .Name
+   Install-XDP
+
+ .Synopsis
+   Installs the eXpress Data Path for Windows service.
+
+ .Description
+   Returns TRUE if the eXpress Data Path for Windows service is installed successfully, otherwise FALSE.
+
+ .Parameter LocalPath
+   Local directory to the eXpress Data Path for Windows service binaries.
+   Default location is $env:LocalAppData\Temp
+
+ .Example
+   # Install the eXpress Data Path service
+   Install-XDP -LocalPath:"$env:TEMP"
+#>
+Function Install-XDP
+{
+   [cmdletbinding(DefaultParameterSetName='Default')]
+
+   Param
+   (
+      [Parameter(ParameterSetName='Default',Mandatory=$false)]
+      [ValidateScript({Test-Path $_ -PathType:'Container'})]
+      [String] $LocalPath = "$env:TEMP"
+   )
+
+   [Boolean] $isSuccess = $true
+
+   Try
+   {
+      If(Assert-SoftwareInstalled -SoftwareName:'XDP for Windows' -Silent)
+      {
+         Write-Host 'XDP for Windows is already installed'
+         return $isSuccess
+      }
+
+      # Download XDP-for-Windows.
+      Write-Host 'Installing eXpress Data Path for Windows'
+      $packageXdpUrl = "https://github.com/microsoft/xdp-for-windows/releases/download/v1.1.0%2Bbed474a/bin_Release_x64.zip"
+      Invoke-WebRequest -Uri $packageXdpUrl -OutFile "$LocalPath\bin_Release_x64.zip"
+      Expand-Archive -Path "$LocalPath\bin_Release_x64.zip" -DestinationPath "$LocalPath\bin_Release_x64" -Force
+      copy "$LocalPath\bin_Release_x64\amd64fre\xdp.cer" $LocalPath
+      copy "$LocalPath\bin_Release_x64\amd64fre\xdpcfg.exe" $LocalPath
+      CertUtil.exe -addstore Root "$LocalPath\xdp.cer"
+      CertUtil.exe -addstore TrustedPublisher "$LocalPath\xdp.cer"
+      Invoke-WebRequest -Uri "https://github.com/microsoft/xdp-for-windows/releases/download/v1.1.0%2Bbed474a/xdp-for-windows.1.1.0.msi" -OutFile "$LocalPath\xdp-for-windows.1.1.0.msi"
```

Comment:
Similar comment to ebpf here - we just need to be mindful that this code contains the specific URL for download